config/locales/en.yml

```diff
@@ -153,12 +153,24 @@ en:
       draft_text:
         one: Draft text
         other: Draft texts
+      edition:
+        one: Edition
+        other: Editions
+      edition_with_appointment:
```

Comment:
Do we need to add translations for these new locales?

Reply:
The changes to this file are all because those formats previously relied on `Edition`'s default behaviour; they now need a name explicitly declared. As I understand it, it'll fall back to the English translation if others don't exist, and so they should be no worse off than prior to this change (where the English name would have been returned in all cases).

Adding translated versions of these names will be part of [this card](https://trello.com/c/kYXvexWo/2455-add-missing-keys-to-locale-files-3).

Reply:
Awesome, I was just checking the translations for these new locales would be picked up at some point. 👍